js/chatbot/shared/Component.svelte

```diff
@@ -39,19 +39,21 @@
 		on:load
 	/>
 {:else if type === "audio"}
-	<svelte:component
-		this={components[type]}
-		{value}
-		show_label={false}
-		show_share_button={true}
-		{i18n}
-		label=""
-		waveform_settings={{ autoplay: props.autoplay }}
-		waveform_options={{}}
-		show_download_button={allow_file_downloads}
-		{display_icon_button_wrapper_top_corner}
-		on:load
-	/>
+	<div style="position: relative;">
```

Comment:
Needed otherwise the audio download buttons pop out of the box

Before

<img width="545" alt="Screenshot 2024-12-17 at 11 45 37 AM" src="https://github.com/user-attachments/assets/7b019ad3-b906-469e-8ed6-835e83ba3de7" />

After
<img width="483" alt="Screenshot 2024-12-17 at 11 45 49 AM" src="https://github.com/user-attachments/assets/c2a648cc-7867-4701-be79-b558054f0ac5" />